{"id":"gcloud/error","name":"Error","title":["Gcloud","Error"],"description":"<p>Base Gcloud exception class.</p>","source":"lib/gcloud/errors.rb#L19","resources":[],"examples":[],"methods":[{"id":"inner-instance","type":"instance","name":"inner","title":["Gcloud","Error#inner"],"description":"<p>The inner error object.</p>","source":"lib/gcloud/errors.rb#L22","resources":[],"examples":[],"params":[],"exceptions":[],"returns":[]},{"id":"initialize-constructor","type":"constructor","name":"initialize","title":["Gcloud","Error#initialize"],"description":"<p>Create a new error object.</p>","source":"lib/gcloud/errors.rb#L26","resources":[],"examples":[],"params":[],"exceptions":[],"returns":[{"types":["<a data-custom-type=\"gcloud/error\">Error</a>"],"description":"a new instance of Error"}]}]}